src/setup-page-script.ts

```diff
@@ -455,4 +500,65 @@ async function __test(storyId: string): Promise<any> {
   });
 }
 
+function expectToHaveNoViolations(results: any): { long: string; short: string } | null {
+  let violations = filterViolations(
+    results.violations,
+    // `impactLevels` is not a valid toolOption but one we add to the config
+    // when calling `run`. axe just happens to pass this along. Might be a safer
+    // way to do this since it's not documented API.
+    results.toolOptions?.impactLevels ?? []
+  );
+
+  function reporter(violations: any) {
+    if (violations.length === 0) {
+      return null;
+    }
+
+    let lineBreak = '\n\n';
+    let horizontalLine = '\u2500\u2500\u2500\u2500\u2500\u2500\u2500\u2500';
+
+    return violations
+      .map((violation: any) => {
+        let errorBody = violation.nodes
+          .map((node: any) => {
+            let selector = node.target.join(', ');
+            let expectedText =
+              red(`Expected the HTML found at $('${selector}') to have no violations:`) + lineBreak;
+            return (
+              expectedText +
+              grey(node.html) +
+              lineBreak +
+              red(`Received:`) +
+              lineBreak +
+              red(`"${violation.help} (${violation.id})"`) +
+              lineBreak +
+              yellow(node.failureSummary) +
+              lineBreak +
+              (violation.helpUrl
+                ? red(`You can find more information on this issue here:`) +
+                  `\n${blue(violation.helpUrl)}`
+                : '')
```

Comment:
Why not `join(lineBreak)` these as well?

Reply:
Because the lineBreak is two \n and I needed only one